AndroidTest/app/src/main/java/cloudant/com/androidtest/keyprovider/ProviderTestUtil.java

```diff
@@ -0,0 +1,28 @@
+package cloudant.com.androidtest.keyprovider;
+
+import com.cloudant.sync.datastore.encryption.KeyData;
+
+import java.security.SecureRandom;
+import java.util.Random;
+import java.util.UUID;
+
+public class ProviderTestUtil {
+    public static String getUniqueIdentifier() {
+        return "test-id-" + UUID.randomUUID();
+    }
+
+    public static KeyData createKeyData() {
+        byte[] encryptedDPK = new byte[32];
+        byte[] salt = new byte[32];
+        byte[] iv = new byte[32];
```

Comment:
Comment addressed in commit https://github.com/cloudant/sync-android/commit/579b63fedb438adf05aafb626e8edd55adb86b59